minecraft_servers/potatonetwork/manifest.json

```diff
@@ -0,0 +1,28 @@
+{
+  "server_name": "potatonetwork",
+  "nice_name": "Potato Network",
+  "direct_ip": "mc.potatonetwork.de",
+  "server_wildcards": [
+    "%.potatonetwork.de"
+  ],
+  "social": {
+    "web": "https://potatonetwork.de/",
```

Comment:
Please remove.
1. Invalid SSL certificate
2. The link is a redirect to LABY.net - It would be a weird redirect on LABY.net

Reply:
Hello, can i remove the Web Link then? Because i don't have an Website yet.

Reply:
Of course.

Reply:
Oh, thanks i will do it